bevy_rapier2d/Cargo.toml

```diff
@@ -42,7 +42,7 @@ log = "0.4"
 serde = { version = "1", features = [ "derive" ], optional = true}
 
 [dev-dependencies]
-bevy = { version = "0.11", default-features = false, features = ["x11"]}
+bevy = { version = "0.12", default-features = false, features = ["x11", "tonemapping_luts"]}
```

Comment:
Not as important, but I don't think we need to include "tonemapping_luts" as part of dev dependencies?

Reply:
Looks like it's not necessary for 2d.